dashboard-for-web/articles/web-dashboard-designer-mode/provide-data/working-with-sql-data-sources/stored-procedures.md

```diff
@@ -4,7 +4,7 @@ author: Natalia Kazakova
 legacyId: 117971
 ---
 # Stored Procedures
-If you use a stored procedure to supply the dashboard with data, you should specify the stored procedure parameters. In the [Dashboard Data Source Wizard](../dashboard-data-source-wizard.md) dialog, select the required stored procedure and click **Next**.
+If you use a stored procedure to supply the dashboard with data, you should specify the stored procedure parameters. In the [Dashboard Data Source Wizard](../../ui-elements/dialogs-and-wizards/dashboard-data-source-wizard.md) dialog, select the required stored procedure and click **Next**.
 
 ![wdd-stored-procedure](../../../../images/img124982.png)
 
```

Comment:
1) Line 15: The following parameter settings are available.

Please change the period to a colon.

2) Last sentence on line 18: See my similar comment above.